### PR TITLE
Image templates icon

### PR DESCRIPTION
**Pull request by Ana06**
Find better icon for image templates and then fix styling of the icon on main page. :bowtie: 

I also removed the width and height as they are overwritten in the css.

## Main page

image

## Image templates page

image

In the image templates page the image is blurred, as its original size is 32x32 and we need it to be 48x48 here. I am not sure if it is a problem, as it is only used to indicate that there is no icon for that template I think it is not that bad. @hennevogel is it possible to have this icon bigger?

### CODE REVIEW

**Comment by Ana06** [comment]
[(8) Present kiwi templates](https://trello.com/c/0JyzpRkk/112-8-present-kiwi-templates)

File: src/api/app/views/webui/image_templates/index.html.haml [hunk 1]
```diff
@@ -13,9 +13,9 @@
             %dd
               %span.form-radio
               - if template.has_icon?
-                %img{ alt: template.title, src: template.source_path('_icon'), width: 64, height: 64 }
+                %img{ alt: template.title, src: template.source_path('_icon') }
               - else
-                = image_tag('distributions/kiwi.png', width: 64, height: 64)
+                = image_tag('image-template.png')
```
Comment (hennevogel) [inline src/api/app/views/webui/image_templates/index.html.haml]: Please also use the sprite.

Comment (Ana06) [inline src/api/app/views/webui/image_templates/index.html.haml]: But the problem is that it hasn't got the same size, that's why we were not using it here. I would say that I need the image in a bigger size and add it also to the sprite.

image

**Comment by hennevogel** [comment]
Just found out, we need to use some other icon set...

**Pull request closed without merging**